website_event_attendee_fields/static/description/index.html

```diff
@@ -0,0 +1,176 @@
+				  <div class="oe_row oe_spaced">
+					<div class="oe_span12">
+					  <p class="oe_mt32">
+						</p><div class="alert alert-info" style="padding:8px;font-weight: 300; font-size: 20px;">
+						  <i class="fa fa-hand-o-right"></i><b> Key features: </b>
+							<ul class="list-unstyled">
+							  <li>
+								<i class="fa fa-check-square-o text-primary"></i>
+								Autocomplete fields when registering an authorized member
+							  </li>
+							  <li>
+								<i class="fa fa-check-square-o text-primary"></i>
+								Styling the registration form with a large number of fields
+							  </li>
+							  <li>
+								<i class="fa fa-check-square-o text-primary"></i>
+								Warning of re-registration
+							  </li>
+							  <li>
+								<i class="fa fa-check-square-o text-primary"></i>
+								Auto-registration of the name and telephone number of the registration to the available values from Partner record
+							  </li>
+							  <li>
+								<i class="fa fa-check-square-o text-primary"></i>
+								Import Attribute Values
+							  </li>
+							
+							</ul>
+						</div>
+					
+					</div>
+				  </div>
+				</section>
+
+				<section class="oe_container oe_dark">
+				  <div class="oe_row oe_spaced">
+					<h3 class="oe_slogan">How It Works</h3>
+					<h5 class="oe_slogan" style="font-size:16px;">
+								You can install this module in a <a title="How to install/update module" href="https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html">usual way</a>. After installation you need to configure each Event. Otherwise there will be no fields at website attendees form.
+					</h5>
+					<div class="oe_span12 text-center">
+					  <p class="oe_mt32">
+						<font size="3px">Go to <em>Events menu → Create new Event</em> menu and add Event.<font>
+					  </font></font></p>		
+					</div>
+					<div class="oe_row_img oe_centered">
+					  <img class="oe_demo oe_picture oe_screenshot" src="events-custom-fields-3.png">
+					</div>
+				   </div>
+				</section>
+
+				<section class="oe_container">
+				  <div class="oe_row oe_spaced">
+					<div class="oe_span12 text-center">
+					  <p class="oe_mt32">
+						<font size="3px">On the site, open the page ``/event``, select the desired Event, specify the number of tickets and click ``[Order Now]``
+						</font>
+					  </p>		
+					</div>
+					<div class="oe_row_img oe_centered">
+					  <img class="oe_demo oe_picture oe_screenshot" src="events-custom-fields-4.png">
+					</div>
+				   </div>
+				</section>	
+
+				<section class="oe_container oe_dark">
+				  <div class="oe_row oe_spaced">
+					<div class="oe_span12 text-center">
+					  <p class="oe_mt32">
+						<font size="3px">As a result, you will see a form with the given fields. Fill in the information</font>
+					  </p>		
+					</div>
+					<div class="oe_row_img oe_centered">
+					  <img class="oe_demo oe_picture oe_screenshot" src="events-custom-fields-6.png">
```

Comment:
Start description with this picture (but use fields set differ from module website_event_attendee_fields_custom) and then   add screenshot of Event form.